msbuild/Xamarin.Mac.Tasks/Xamarin.Mac.Common.targets

```diff
@@ -727,15 +729,25 @@ Copyright (C) 2014 Xamarin. All rights reserved.
 
 	<Target Name="_CopyAppExtensionsToBundle"  Condition="'$(IsAppExtension)' != 'true'" DependsOnTargets="_ResolveAppExtensionReferences">
 		<MakeDir Directories="$(_AppBundlePath)Contents\PlugIns" Condition="'@(_ResolvedAppExtensionReferences)' != ''" />
+		<MakeDir Directories="$(_AppBundlePath)Contents\XPCServices" Condition="'@(_ResolvedAppExtensionReferences)' != ''" />
```

Comment:
Would this mean every Xamarin.Mac app now ends up with an XPCServices directory?

Reply:
As it currently exists, yes. However, AFAIK an empty XPCServices directory does no harm. I'll try adding `Condition="'%(_ResolvedAppExtensionReferences.Extension)' == '.xpc'"`.